backoffice/modules/catalog/components/DisplayTypeModel.tsx

```diff
@@ -0,0 +1,134 @@
+import { ProductOption } from '@catalogModels/ProductOption';
+import React from 'react';
+import { Modal, Button, Form, Row, Col } from 'react-bootstrap';
+import { SingleValue } from 'react-select';
+
+type DisplayTypeModalProps = {
+  show: boolean;
+  handleCloseModel: () => void;
+  handleColorChange: (event: React.ChangeEvent<HTMLInputElement>, optionValueKey: string) => void;
+  currentModelOption: SingleValue<ProductOption>;
+  productOptionValuePost: {
+    productOptionId?: number;
+    value: { [key: string]: string };
+    displayType?: string;
+    displayOrder?: number;
+  }[];
+  setDisplayType: (type: string) => void;
+};
+
+const DisplayTypeModal: React.FC<DisplayTypeModalProps> = ({
```

Comment:
It looks like there's a typo in the file name for DisplayTypeModal; it’s currently named DisplayTypeModel